### PR TITLE
SCRIPTS/SQL: Added Fruit Parfait/Queen's Crown

### PR DESCRIPTION
**Pull request by Zenny89**
Source: http://www.ffxivpro.com/forum/topic/45007/look-for-food-effects-list-of-food-2014/

### CODE REVIEW

**Comment by Hozu** [comment]
Woah, food with magic attack bonus what.

I checked the synth tables, and there isn't a recipe for this in it. Does one need to do a packet capture to get something first or is the ID arbitrary and the next one can simply be used for a new recipe?

**Comment by Zenny89** [comment]
I wasn't sure. If someone explains which, I'll go ahead and add it to this request.

**Comment by Hozu** [comment]
[18:56:59]	Hozu	that reminds me
[18:57:11]		Hozu	is a packet capture required for adding synths for any reason?
[19:08:41]	R3P0FFXI	Hozu thats just recipies
[19:09:19]	Hozu	so nothing special about adding recipes? Just fill in the correct information using the next open ID?
[19:09:47]	R3P0FFXI	nope

Unless someone says otherwise I'd go add the recipe to this commit at this point.

**Comment by Zenny89** [comment]
Updated with recipe.

File: scripts/globals/items/fruit_parfait.lua [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+-----------------------------------------
+
+require("scripts/globals/status");
+
+-----------------------------------------
+-- OnItemCheck
+-----------------------------------------
+
+function onItemCheck(target)
+local result = 0;
```
Comment (TeoTwawki) [inline scripts/globals/items/fruit_parfait.lua]: Indent missing. same with the return.

File: scripts/globals/items/queens_crown.lua [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+-----------------------------------------
+-- OnItemCheck
+-----------------------------------------
+
+function onItemCheck(target)
+local result = 0;
+    if (target:hasStatusEffect(EFFECT_FOOD) == true or target:hasStatusEffect(EFFECT_FIELD_SUPPORT_FOOD) == true) then
+        result = 246;
+    end
+return result;
```
Comment (TeoTwawki) [inline scripts/globals/items/queens_crown.lua]: This file too

**Comment by Zenny89** [comment]
Updated

**Pull request merged**